Packagehub Admonition

Loquacity opened the pull request:
# Description

Based on a conversation from the Core team call 2020-10-27, adds a note about phub being big, and not to enable it unless required.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes #<insert issue or PR link, if any>

paususe commented on modules/client-configuration/pages/channels.adoc:
```diff
@@ -33,6 +33,12 @@ This provides the {phub} channels, which you can subscribe your clients to.
 
 On Salt clients, you must apply the highstate after subscribing to the channel to be able to install packages.
```
This sentence should be above the PackageHub section, this is true for all Salt clients.

paususe commented on modules/client-configuration/pages/channels.adoc:
```diff
@@ -33,6 +33,12 @@ This provides the {phub} channels, which you can subscribe your clients to.
 
+[NOTE]
+====
+{phub} provides a large number of packages, which can take a long time to synchronize.
```
```suggestion
{phub} provides a large number of packages, which can take a long time to synchronize and take a large amount of disk space. {phub} packages receive only community support and are not supported by SUSE.
```

paususe commented on modules/client-configuration/pages/channels.adoc:
```diff
@@ -33,6 +33,12 @@ This provides the {phub} channels, which you can subscribe your clients to.
+[NOTE]
+====
+{phub} provides a large number of packages, which can take a long time to synchronize.
+Do not enable it unless you require the packages it provides.
```
```suggestion
To avoid unintended updates or installations of software from a non-SUSE supported source, it is recommended to establish a Content Lifecycle Management strategy based on denying all {phub} packages first and then explicitly enabling only the specific packages you require.
```

paususe reviewed (approved):
A few suggestions.

Also, the next section (AppStreams) is wrong. It is conflating two different things: EPEL and appstreams. EPEL is the equivalent to PackageHub in the RHEL/CentOS/ES/OL world; appstreams are part of the operating system. Please fix that in another PR and I will review.

paususe reviewed (changes requested):
Oops, approved by mistake, I actually wanted to request changes

Loquacity commented:
> A few suggestions.

Added the suggestions and moved things around a little. Hopefully that makes it all a bit clearer.

> 
> Also, the next section (AppStreams) is wrong. It is conflating two different things: EPEL and appstreams. EPEL is the equivalent to PackageHub in the RHEL/CentOS/ES/OL world; appstreams are part of the operating system. Please fix that in another PR and I will review.

Will do :+1:

Loquacity commented:
See https://github.com/uyuni-project/uyuni-docs/pull/647 for EPEL/Appstream

paususe commented on modules/client-configuration/pages/channels.adoc:
```diff
@@ -21,17 +21,31 @@ For example, if a system has been assigned to the {sles}{nbsp}15 `x86_64` base c
+{phub} provides a large number of packages, which can take a long time to synchronize and consume a large amount of disk space.
+Do not enable {phub} unless you require the packages it provides.
+
+To avoid unintentionally installing or updating unsupported packages, we recommended that you implement a content lifecycle management strategy that initially denies all {phub} packages.
```
```suggestion
To avoid unintentionally installing or updating unsupported packages, we recommend that you implement a content lifecycle management strategy that initially denies all {phub} packages.
```

Sorry, for some reason I used the past tense in my previous suggestion :-/

Loquacity replied:
I really should have noticed that :woman_facepalming:

paususe reviewed (changes requested):
Just fix the mistaken use of past tense in my previous suggestion and it's good to merge

Loquacity commented on modules/reference/pages/configuration/files-locally-managed.adoc:
```diff
@@ -3,8 +3,8 @@
-Local files have higher priority than centrally-managed files.
-If a system is subscribed to a configuration channel with a given file and additionally has a locally-managed version of that file, the locally-managed version will be deployed.
+Local files have higher priority than centrally managed files.
+If a system is subscribed to a configuration channel with a given file and additionally has a locally-managed version of that file, the locally managed version will be deployed.
```
```suggestion
If a system is subscribed to a configuration channel with a given file and additionally has a locally managed version of that file, the locally managed version will be deployed.
```
